CHANGELOG.md

```diff
@@ -1,3 +1,10 @@
+### New features
```

Comment:
Change log is generated by a script based on commit messages in angular format. 

Please roll back the changes in this file and change `chore` to `feat` in the commit message.